db/schema.rb

```diff
@@ -11,18 +11,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160411180305) do
+ActiveRecord::Schema.define(version: 20160413135006) do
+
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "landmarks", force: :cascade do |t|
-    t.string   "name",        null: false
-    t.string   "location",    null: false
-    t.string   "image",       null: false
-    t.text     "description", null: false
-    t.datetime "created_at",  null: false
-    t.datetime "updated_at",  null: false
-    t.integer  "user_id",     null: false
+    t.string   "name",                         null: false
+    t.string   "location",                     null: false
+    t.string   "image",                        null: false
+    t.text     "description",                  null: false
+    t.datetime "created_at",                   null: false
+    t.datetime "updated_at",                   null: false
+    t.integer  "user_id",                      null: false
+    t.string   "average_review", default: "0"
```

Comment:
@Salisbum Ok! It would be interesting to know exactly what kind of issues so we could sort through them and get it working. I just am hesitant to agree to using a string for numeric data unless there's a demonstrable benefit/reason.